File: Software Technologies/Exams/Exam-2Sep2017/JavaScript/models/Product.js
const mongoose = require('mongoose');

let productSchema = mongoose.Schema({
	priority : {type: Number, required: true},
    name: {type: String, required: true},
	quantity: {type: Number, required: true},
    status: {type: String, required: true}
});

let Product = mongoose.model('Product', productSchema);

module.exports = Product;

File: Software Technologies/Exams/Exam-2Sep2017/C#/ShoppingList/Models/Product.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace ShoppingList.Models
{
    public class Product
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public int Priority { get; set; }

        [Required]
        [AllowHtml]
        [StringLength(50)]
        public string Name { get; set; }

        [Required]
        public int Quantity { get; set; }

        [Required]
        [AllowHtml]
        public string Status { get; set; }
    }
}